http2/pom.xml

```diff
@@ -1,210 +1,293 @@
-                    </execution>
-                </executions>
-                <configuration>
-                    <artifactItems>
-                        <artifactItem>
-                            <groupId>org.mortbay.jetty.alpn</groupId>
-                            <artifactId>alpn-boot</artifactId>
-                            <version>${alpn-boot.version}</version>
-                            <type>jar</type>
-                            <overWrite>false</overWrite>
-                        </artifactItem>
-                    </artifactItems>
-                    <outputDirectory>${project.build.directory}/test-lib</outputDirectory>
-                    <stripVersion>true</stripVersion>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.21.0</version>
-                <configuration>
-                    <argLine>-Xbootclasspath/p:${project.build.directory}/test-lib/alpn-boot.jar</argLine>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
-    <profiles>
-        <profile>
-            <id>jdk8_121</id>
-            <activation>
-                <jdk>1.8.0_121</jdk>
-            </activation>
-            <properties>
-                <alpn-boot.version>8.1.11.v20170118</alpn-boot.version>
-            </properties>
-        </profile>
-        <profile>
-            <id>jdk8_131</id>
-            <activation>
-                <jdk>1.8.0_131</jdk>
-            </activation>
-            <properties>
-                <alpn-boot.version>8.1.11.v20170118</alpn-boot.version>
-            </properties>
-        </profile>
-        <profile>
-            <id>jdk8_141</id>
-            <activation>
-                <jdk>1.8.0_141</jdk>
-            </activation>
-            <properties>
-                <alpn-boot.version>8.1.11.v20170118</alpn-boot.version>
-            </properties>
-        </profile>
-        <profile>
-            <id>jdk8_144</id>
-            <activation>
-                <jdk>1.8.0_144</jdk>
-            </activation>
-            <properties>
-                <alpn-boot.version>8.1.11.v20170118</alpn-boot.version>
-            </properties>
-        </profile>
-        <profile>
-            <id>jdk8_151</id>
-            <activation>
-                <jdk>1.8.0_151</jdk>
-            </activation>
-            <properties>
-                <alpn-boot.version>8.1.11.v20170118</alpn-boot.version>
-            </properties>
-        </profile>
-        <profile>
-            <id>jdk8_152</id>
-            <activation>
-                <jdk>1.8.0_152</jdk>
-            </activation>
-            <properties>
-                <alpn-boot.version>8.1.11.v20170118</alpn-boot.version>
-            </properties>
-        </profile>
-    </profiles>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Please, remove code style changes. You have diff +3000 lines and -3200 lines . It's impossible for review